### ISSUE TITLE
Navigation hotspot jumps to start when Destination is not set

### ISSUE THREAD

**Issue by RogueAgent991** [h5p/moodle-mod_hvp#504]
When a Navigation hot spot is used to pause the video, I have to enter a Destination timecode. Without that, the hot spot restarts the video. It would be consistent with the other interactions if the hot spot continued from it's current location if no destination is entered.

TIA

**Comment by otacke** [comment]
@RogueAgent991 Hi! I was made a community maintainer for H5P Group's repositories a few weeks ago, and I am still in the process trying to tackle all the open issues and pull requests.

I am terribly sorry that you have not received a reply before.

I can reproduce your finding. Undefined values are always a hot topic - how to interpret them? I'd assume that some though must have gone into this, or the field would not have been declared optional. Regarding a solution, I'd even go so far as to not display the `Navigation Hotspot` at all if no destination is entered. But I think that either solution (just continuing or not showing the hotspot to begin with) would be better than to simply restart the video. Agreed!

I'll move this issue to "verified by community maintainer", then H5P Group will have to decide whether this works as intended or what other behavior should be implemented.

**Comment by RogueAgent991** [comment]
I will take the opportunity to restate my recommendation: if no Destination timecode is entered, restart the video from it's current location.

Doing so would allow me to copy & paste a "Continue" button, without having to enter each one and setting a destination timecode.

Thx

**Comment by otacke** [comment]
@RogueAgent991 What's the purpose of adding a Navigation Hotspot without a target? Just a hack to pause the video and allow to continue it?

It will be up to H5P Group to determine if desirable user experience or whether for that purpose a simple "stop video" interaction which additional specific options, e.g. auto-continue after x seconds or whatever, would be better suited (clearer intent) to achieve what yo want.

**Comment by RogueAgent991** [comment]
@otacke 

When teaching, there are times when you want the learner to do something that does not involve an H5P activity.
Engaging the learner is critical for learning, and not all of these engagements can be using an H5P activity in Interactive Video.

- For example, "Attempt this problem on your own. Click continue once you are done."

ALL our courses use a Navigation Hotspot to pause the video, wanting it to continue from that spot. These examples are from a self-defense course we offer. 

<img width="1868" height="1055" alt="Image" src="https://github.com/user-attachments/assets/2aa37436-9030-4a22-84d1-1fb8137197b8" />

<img width="1870" height="1055" alt="Image" src="https://github.com/user-attachments/assets/1f8d58af-7be4-4952-b9cb-cc86582219e2" />

For the third picture, the audio says, "The Criminal Code is actually quite readable. Most sections are written in plain English. Please take a few seconds to read Section 34, one, of the Criminal Code."

<img width="1869" height="1055" alt="Image" src="https://github.com/user-attachments/assets/d3571da3-57f8-44c3-9302-a63da07d6312" />

**Comment by otacke** [comment]
@RogueAgent991 As I suspected, you'd want to use the navigation activity not for navigation, but for pause/resume. That's what H5P Group has to decide on, because the activity name "navigation hotspot" would not reveal the full spectrum of actions. And as I said, it might make more sense to add a "pause" activity for that very same purpose, which could then have specific options (e.g. rules whether to pause or not, auto-continue timer, etc.). The latter would make the intent clear and allow not to mix options.
